subprojects/docs/src/samples/java/multiproject/kotlin/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ subprojects {
 
     version = "1.0"
 
-    tasks.getByName<Jar>("jar") {
+    tasks.jar {
```

Comment:
:x: should be `tasks.named<Jar>("jar")`, type-safe accessors aren't available for configuration injection